rescheduler/rescheduler.go

```diff
@@ -146,10 +150,10 @@ func main() {
 					continue
 				}
 
-				criticalPods := filterCriticalPods(allUnschedulablePods, podsBeingProcessed)
+				criticalPodsCreatedByDaemonSets := filterCriticalPodsCreatedByDaemonSets(allUnschedulablePods, podsBeingProcessed)
```

Comment:
optional: s/criticalPodsCreatedByDaemonSets/criticalDaemonSetPods/ to be shorter